simple-git/typings/simple-git.d.ts

```diff
@@ -16,7 +16,7 @@ export interface SimpleGitBase {
    /**
     * Adds one or more files to source control
     */
-   add(files: string | string[], callback?: types.SimpleGitTaskCallback<string>): Response<string>;
+   add(files: string | string[], force: bool = false, callback?: types.SimpleGitTaskCallback<string>): Response<string>;
```

Comment:
This would be a breaking change, why not introducing an optional `options` after the `callback` instead?